arcgis-ios-sdk-samples/Maps/Show magnifier/README.md

```diff
@@ -1,14 +1,33 @@
 # Show magnifier
 
-This sample demonstrates how you can tap and hold on a map to get the magnifier. You can also pan while tapping and holding to move the magnifier across the map.
+Tap and hold on a map to show a magnifier.
 
-![](image1.png)
+![Image of show magnifier](show-magnifier.png)
+
+## Use case
+
+Due to the limited screen size of some mobile devices, it may be difficult to identify individual features on a map where there is a high density of information, or the scale is very small. This can be the case when a mobile device is used for navigation, and the user wishes to magnify a particular area to better identify a road intersection.
```

Comment:
```suggestion
Due to the limited screen size of some mobile devices, it may be difficult to identify individual features on a map where there is a high density of information or the scale is very small. This can be the case when a mobile device is used for navigation and the user wishes to magnify a particular area to better identify a road intersection.
```